wazo_calld/plugins/transfers/state.py

```diff
@@ -526,40 +593,18 @@ def initiator_hangup(self):
 
     @transition
     def initiator_joined_stasis(self):
-        bridge = self._ari.bridges.get(bridgeId=self.transfer.id)
-        bridge.addChannel(channel=self.transfer.initiator_call)
-
-        try:
-            context = self._ari.channels.getChannelVar(
-                channelId=self.transfer.initiator_call,
-                variable='XIVO_TRANSFER_RECIPIENT_CONTEXT',
-            )['value']
-            exten = self._ari.channels.getChannelVar(
-                channelId=self.transfer.initiator_call,
-                variable='XIVO_TRANSFER_RECIPIENT_EXTEN',
-            )['value']
-            variables_str = self._ari.channels.getChannelVar(
-                channelId=self.transfer.initiator_call,
-                variable='XIVO_TRANSFER_VARIABLES',
-            )['value']
-            timeout_str = self._ari.channels.getChannelVar(
-                channelId=self.transfer.initiator_call, variable='XIVO_TRANSFER_TIMEOUT'
-            )['value']
-        except ARINotFound:
-            logger.error('initiator hung up while creating transfer')
-        try:
-            variables = json.loads(variables_str)
-        except ValueError:
-            logger.warning('could not decode transfer variables "%s"', variables_str)
-            variables = {}
-        timeout = None if timeout_str == 'None' else int(timeout_str)
+        # TODO: any error to interpret?
```

Comment:
I guess `get_initial_transfer_variables` should raise an error if it can't get all variables. Since this case was not handled before either, I don't think it's very important for now.